book/src/dev/release-criteria.md

```diff
@@ -0,0 +1,57 @@
+# Go/No-Go Release Criteria
+
+## Alpha Release
+
+In the first alpha release, we want Zebra to participate in the network and replicate the Zcash chain state. We also want to validate proof of work and the transaction merkle tree, and serve blocks to peers.
+
+### System Requirements
+
+_TBD_
```

Comment:
@dconnolly can you copy the CI configs into this suggestion?

```suggestion
Our CI tests that Zebra builds, passes its tests, and syncs to tip on the following configurations:
...

Our CI tests that Zebra builds and passes its tests on the following configurations:
...
```

Reply:
I've un-resolved this conversation, because @dconnolly hasn't filled in this info yet.